install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -38,6 +38,76 @@ separately in the following Red Hat Gluster Storage documentation:
 
 link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html/container-native_storage_for_openshift_container_platform/[Container-Native Storage for OpenShift Container Platform]
 
+[[container-native-storage-recommendations]]
+=== Container Native Storage Recommendations
+
+{product-title} offers container native storage (CNS) storage, which makes it
+easier for {product-title} users to fulfill their storage needs. With CNS,
+solution users and administrators are empowered to have storage and application
+pods running together on the same infrastructure and sharing the same resources.
+
+Follow the
+link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html/container-native_storage_for_openshift_container_platform/[Container
+Native Storage with GlusterFS] guidance for configuring CNS as part of a
+{product-title} cluster.
+
+Currently, up to 100 CNS volumes on one trusted storage pool (TSP) is supported.
+
+[[volumes-creation-time-for-container-native-storage]]
+==== Creation Time of Volumes with Container Native Storage
+
+When deploying applications on {product-title}, it is important to estimate how
+long it will take to build an environment, in regard to storage, so that an
+application pod is able to start.
+
+If the application pod requires a persistent volume claim (PVC), then you must
+consider the time it takes for a different number of CNS to be created and bound
+to the corresponding PVC.
```

Comment:
I'm a little confused about the above two paras. Can I suggest something like: 

"Building environment storage can influence the time it takes for an application to start. For example, if the application pod requires a persistent volume claim (PVC), then extra time might have to be considered for CNS to be created and bound to the corresponding PVC. This effects the build time for an application pod to start."